demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -312,3 +339,97 @@ def execute_command(self, command: str):
         context = ast.literal_eval(context)
 
         return res, context
+
+    def get_server_type(self) -> ServerType:
+        if not self.client.api_client.configuration.host.startswith("https://api-"):
+            return ServerType.XSOAR
+        raw_response, _, response_headers = self.client.generic_request(
+            "/about", "GET", response_type="object"
+        )
+        if "text/html" in response_headers.get("Content-Type"):
+            raise ValueError(
+                "",  # TODO
```

Comment:
We should raise a more informative message here, and remove the #TODO comment :)